Integrations/QRadar/QRadar.yml

```diff
@@ -40,15 +40,15 @@ configuration:
   name: incidentType
   required: false
   type: 13
-- display: Full Incident Enrichment
-  defaultvalue: 'true'
+- defaultvalue: 'true'
+  display: Full Incident Enrichment
   hidden: false
   name: full_enrich
   required: false
   type: 8
 description: Fetch offenses as incidents and search QRadar
-display: IBM QRadar
-name: QRadar
+display: IBM QRadar 5.5
+name: IBM QRadar 5.5
```

Comment:
same here

Reply:
the original ID was IBM QRadar,
Do you want me to remove 5.5 ?